backend/src/controllers/shiftController.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from 'express';
+import ShiftModel from '../models/shiftModel.js';
+
+export default class ShiftController {
+
+	public async getShiftInfo(req: Request, res: Response, fk_volunteer_id:string, fk_schedule_id:number, shift_date:string){
```

Comment:
since we have the req: Request here, lets access the fields in the controller instead of passing as params:

getShiftInfo(req: Request, res: Response)
    const fk_volunteer_id: string = req.body.volunteerID;
    ... etc
and then also throw an error if any fields are missing